File: 프로그래머스/lv1/12930. 이상한 문자 만들기/이상한 문자 만들기.cpp
#include <string>
#include <vector>
#include <ctype.h>
using namespace std;

string solution(string s) {
    string answer = s;
    for (int i = 0, cnt = 0; i < s.length(); i++, cnt++)
    {
        if (cnt % 2 == 0 && islower(answer[i]))
        {
            answer[i] -= 32;
        }
        else if (cnt % 2 == 1 && isupper(answer[i]))
        {
            answer[i] += 32;
        }
        else if (answer[i] == ' ')
        {
            cnt = -1;
        }
    }

    return answer;
}

File: 프로그래머스/lv1/12930. 이상한 문자 만들기/이상한 문자 만들기.cs
public class Solution {
    public string solution(string s) {
    string answer = "";
    int wordIndex = 0;
    for (int i = 0; i < s.Length; i++)
    {
        if (s[i] == ' ')
        {
            answer += s[i];
            wordIndex = 0;
            continue;
        }
        if (wordIndex % 2 == 0)
        {
            answer += s[i].ToString().ToUpper();
            wordIndex++;
        }
        else
        {
            answer += s[i].ToString().ToLower();
            wordIndex++;
        }
    }
    return answer;
    }
}